### PR TITLE
fix: storybook local environment

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/regen-web#1974

fix:
- infinite restart issue on local environment 63758d4b907234cac4b1762455caf47d82dca4b4
- Buffer issue in some stories 345f64888184a2ffea1ed2f48b8599abe2511d1e
- marketplace stories (previously disabled) 8f653e4bbb2cd3b0a2737c74bd914a01db00afd3

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: .yarn/patches/fsevents-npm-1.2.13-9e970219e7.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff --git a/fsevents.js b/fsevents.js
+index 939cc7e3d177dd2d4f889f8e97a5b7d3c59fc68f..e469059ac6defb501e860e549faefc306ec12bac 100644
+--- a/fsevents.js
++++ b/fsevents.js
+@@ -73,6 +73,7 @@ function proxies(ctor, target) {
+ function getFileType(flags) {
+   if (Native.Constants.kFSEventStreamEventFlagItemIsFile & flags) return 'file';
+   if (Native.Constants.kFSEventStreamEventFlagItemIsDir & flags) return 'directory';
++  if (Native.Constants.kFSEventStreamEventFlagMustScanSubDirs & flags) return 'directory';
```
Comment (flagrede) [inline .yarn/patches/fsevents-npm-1.2.13-9e970219e7.patch:9]: For more information about this change see this issue: https://github.com/vitejs/vite/issues/14043
The storybook restart issue in local was related to the dependency `fsevents` used in `chodikar` (the file watcher used in storybook). The issue has been fixed in the latest version of `fsevents` but this version seems incompatible on linux atm so we're patching the dependency with the fix in the meantime.

File: patches/fsevents+2.3.2.patch [hunk 2]
```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/fsevents/fsevents.js b/node_modules/fsevents/fsevents.js
+index f1b31c9..a75636e 100644
+--- a/node_modules/fsevents/fsevents.js
++++ b/node_modules/fsevents/fsevents.js
+@@ -50,6 +50,7 @@ function getInfo(path, flags) {
+ function getFileType(flags) {
+   if (events.ItemIsFile & flags) return "file";
+   if (events.ItemIsDir & flags) return "directory";
++  if (events.MustScanSubDirs & flags) return "directory";
```
Comment (flagrede) [inline patches/fsevents+2.3.2.patch:9]: For more information about this change see this issue: https://github.com/vitejs/vite/issues/14043
The storybook restart issue in local was related to the dependency `fsevents` used in `chodikar` (the file watcher used in storybook). The issue has been fixed in the latest version of `fsevents` but this version seems incompatible on linux atm so we're patching the dependency with the fix in the meantime.

**Comment by flagrede** [comment]
There's still an issue with the wallet connection.

**Comment by flagrede** [comment]
Issue mentioned above was not related to the PR, will be fixed here: https://github.com/regen-network/regen-web/pull/2058

**Comment by flagrede** [comment]
New fix here: https://github.com/regen-network/regen-web/pull/2066

**Comment by flagrede** [comment]
@regen-network/rnd-dev-team the PR is ready for review again.

**Review by blushi — APPROVED** [review]
Storybook works back again normally in local dev, thanks!

**Pull request merged**